### ISSUE TITLE
Politically biased source

### ISSUE THREAD

**Issue by vrozsas13** [StevenBlack/hosts#1190]
Is there any reason for you to still be using a politically biased source for the fake news extension?
A lot of right-wing leaning news outlets are being blocked, while left-wing leaning websites with a long publicly known history of publishing fake news are not blocked (like CNN, since it's report on Saddam's supposed WMDs, it consistently lowered it's bar and became more of a propaganda tool than actual news outlet).
And JihadWatch.org? Really?

There's no objective universal method in marktron's project to decide whether a domain will enter the list; the decision seems pretty arbitrary and highly susceptible to bias from the maintainer's part.

An additional point (a lot more philosophical and doesn't address the issue itself, but it's still worth mentioning): Even if the list wasn't as biased as it is now and only contained actual fake news spreaders, I highly doubt that anyone would need it. One should use their critical thinking to decide on whether to believe a news story or not.
Using content blocking this way makes people's brains lazy. Now people will depend on technology even to think for them? That's absurd. Rational thinking is one of the main things that separate humans from the other animals.
People shouldn't live in bubbles/ecochambers. People should read/hear both sides of the debate, question them, and then come to a conclusion (by themselves) on what to believe or support.

**Comment by llacb47** [comment]
If you don't like the list, you don't have to use it.

**Comment by vrozsas13** [comment]
> If you don't like the list, you don't have to use it.

Thanks for the suggestion, but I'm not using it already.
I don't need to, I have a brain, I can think for myself.

**Comment by vrozsas13** [comment]
> If you don't like the list, you don't have to use it.
> 
> Edit: `https://en.wikipedia.org/wiki/Jihad_Watch`

You're really using wikipedia as your reference/source? They cite ADL, that considers anything to be a hate symbol and everyone to be spreading hate.
The ADL is not only biased, it's also radical.

Jihad Watch explicitly excludes "Muslims"  (as a generic term) from the issue. Their focus is on the extremists ("violent jihad" - as they put it, since there's other forms of jihad and they seem to acknowledge).

You never read it and still, you're judging something that you don't know?
That was exactly what I complained about in the original comment...

**Comment by vrozsas13** [comment]
I'm reposting my last comment but this time I excluded a whole paragraph where I committed a phalacy (sorry for my stupidity). Below follows my previous comment (deleted) minus the phalacy:

> Edit: `https://en.wikipedia.org/wiki/Jihad_Watch`

Jihad Watch explicitly excludes "muslims"  (as a generic term) from the issue. Their focus is on the extremists ("violent jihad" - as they put it, since there's other forms of jihad and they seem to acknowledge this fact).

You never read it and still, you're judging it. Well... "you" aren't, but your peers who you follow blindly are.
You're condemning something that you don't even know?
That was exactly what I complained about in the original comment: bias.
